src/integrationTest/resources/ccd/response/search-by-legacy-id/result-format-single-case.json

```diff
@@ -0,0 +1,14 @@
+{
+  "total": 1,
+  "cases": [
+    {
+      "id": %1$s,
```

Comment:
it would be nice to maybe put a readme file to a folder containing those json files explaining this.

Reply:
done